scripts/verify/verify-rf.sh

```diff
@@ -16,6 +16,7 @@ set -x
 "${RF_VENV}/bin/robocop" \
     --exclude 1015 \
     --configure 0504:max_len:40 \
-    --configure 0505:max_calls:20
+    --configure 0505:max_calls:20 \
+    --configure 0508:line_length:200
```

Comment:
I've been trying to keep the line length on 80-100 chars, and most tests are like this.
Maybe, we can introduce a more conservative limit to keep tests easily readable?

Reply:
The command to run subscription manager to enable a repo is longer than 80 characters and trying to split it across lines as a shell command in RF was causing issues.